spec/features/visitor_sees_all_the_user_spec.rb

```diff
@@ -34,7 +34,11 @@
   end
 
   scenario "user sees a maximum of 12 users per page" do
-    15.times { FactoryGirl.create(:user, username: Faker::Name.name) }
+    15.times { FactoryGirl.create(
+      :user,
+      username: Faker::Name.name,
+      phone_number: Faker::Number.number(10)
+      ) }
```

Comment:
Expression at 41, 9 should be on its own line.<br>Indent `)` the same as the start of the line where `(` is.